src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_exception_handler.py

```diff
@@ -0,0 +1,16 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+def build_exception_wrapper(message=None):
```

Comment:
When I look at where this is used in commands.py, it doesn't look like there's any need for this to be a factory.  You can simply remove this and pass "build_exception" directly

`exception_handler=build_exception`

I would also rename build_exception to something more descriptive like `ams_exception_handler`.

Reply:
Good catch, we'll clean and rename this exception handler.

Reply:
Done